### ISSUE TITLE
Dead Link

### ISSUE THREAD

**Issue by maymage** [flathub/edu.stanford.Almond#21]
Hi, 

I could be mistaken, but is Almond not "archived" by now?

Last update 2+ years ago, the link https://almond.stanford.edu/ is dead etc?

**Comment by gcampax** [comment]
Yeah the Almond project was terminated and this app is not functional anymore. I don't know how to archive it and remove it from Flathub though.

**Comment by TingPing** [comment]
It should be marked EOL first.

If it still builds: https://github.com/flathub/flathub/wiki/App-Maintenance#end-of-life

Otherwise an admin can manually mark it.

**Comment by yakushabb** [comment]
@bbhtt Could you mark the application as EOL manually?

The source code appear to be removed.

**Comment by bbhtt** [comment]
Source seems still here https://github.com/stanford-oval/almond-gnome Which module is gone?

**Comment by yakushabb** [comment]
> Yeah the Almond project was terminated and this app is not functional anymore. I don't know how to archive it and remove it from Flathub though.

Source: https://github.com/flathub/edu.stanford.Almond/issues/21#issuecomment-1556916865
by @gcampax

**Comment by bbhtt** [comment]
So the source is still here. I guess you meant it is deprecated by upstream.

If you can get the old version building, then EOL-ing will be easier. Otherwise it needs to be manually EOL-ed on the server.

**Comment by yakushabb** [comment]
Closed, due to EOL.

https://github.com/flathub/edu.stanford.Almond/pull/22
